test/api_spec.js

```diff
@@ -321,6 +325,36 @@ describe("api", function () {
       });
     });
   });
+  describe("backup resource", function () {
+    this.timeout(TIMEOUT.MEDIUM);
+
+    const publicId = "api_test_backup_restore" + SUFFIX;
+    before(() => uploadImage({
+      public_id: publicId,
+      backup: true,
+    }).then(() => cloudinary.v2.api.resource(publicId)).then((resource) => {
+      expect(resource).not.to.be(null);
+    }));
+    after(function () {
+      return cloudinary.v2.api.delete_resources(publicId).then((response) => {
+        expect(response).to.have.property("deleted");
+      });
+    });
+    it("should return the asset details together with all of its backed up versions when versions is true", function () {
+      return cloudinary.v2.api.resource(publicId, { versions: true })
+        .then((resource) => {
+          expect(resource.versions).not.to.be(undefined);
```

Comment:
I guess these are the same no? if 'resource.versions' is an array, it's also not undefined?

Reply:
yeah, I'll remove the undefined one